packages/react-environment/src/webb-context/relayer/webb-relayer.class.ts

```diff
@@ -116,9 +133,9 @@ export class WebbRelayerBuilder {
           : new Map(),
         substrate: info.substrate
           ? Object.keys(info.substrate)
-              .filter((key) => info.substrate[key]?.account && nameAdapter(key, 'evm') != null)
+              .filter((key) => info.substrate[key]?.beneficiary || nameAdapter(key, 'substrate') != null)
```

Comment:
Will the relayer for substrate always return `beneficiary`? Or is it still optional and if a beneficiary is not set in the config, will it return `account`?

Reply:
I believe the `account` the field is deprecated in favor of the `beneficiary`, And relayer that supports substrate will have the `beneficiary` key  not account that's why I just added just `beneficiary` and it wont hurt adding `account` as well if you think so

Reply:
Nah it's better not to support it. Much better idea for it always to be beneficiary. Would you mind just adding a comment above line 127 that account is to support Mainnet Harmony and Shiden relayer deployments?

Reply:
This is why I didn't add it @nepoche! , will restore to what I first implemented and add the comment you suggested